src/Bookings.js

```diff
@@ -1,21 +1,140 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import BookingForm from "./BookingForm.js";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
+import SearchResults from "./SearchResults.jsx";
 // import FakeBookings from "./data/fakeBookings.json";
 
-const Bookings = () => {
+const Bookings = props => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [handleError, setHandleError] = useState(null);
+
+  function handleAddPerson(newBooking) {
+    setBookings([...bookings, newBooking]);
+  }
+
+  // console.log(bookings);
+
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    const filteredValue = bookings.filter(value => {
+      return (
+        value.firstName.toLowerCase().includes(searchVal) ||
+        value.surname.toLowerCase().includes(searchVal)
+      );
+    });
+    setBookings(filteredValue, searchVal);
   };
 
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      .then(response => {
+        return response.json();
+      })
+      .then(data => {
+        // console.log(data);
+        if (data.error) {
+          setHandleError(data.error);
+        } else {
+          setBookings(data);
+        }
+        setLoading(false);
+      })
+      .catch(error => {
+        console.log(error);
```

Comment:
Anything else we might want in our catch function here? We are only logging the error